.github/workflows/docker_build.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     if: |
       github.repository == 'azerothcore/azerothcore-wotlk'
       && !github.event.pull_request.draft
-      && (github.ref_name == 'master' || contains(github.event.pull_request.labels.*.name, 'run-build') || github.event.label.name == 'run-build')
+      && (github.ref == 'refs/heads/master' || contains(github.event.pull_request.labels.*.name, 'run-build') || github.event.label.name == 'run-build')
```

Comment:
Not sure I understand what the change here is doing. It's still comparing the ref name, isn't it?